README.md

```diff
@@ -195,7 +195,8 @@ Finally, in a separate shell, run a model:
 
 ## REST API
 
-> See the [API documentation](docs/api.md) for all endpoints.
+**See the [API documentation](docs/api.md) for all endpoints.**
```

Comment:
Thanks for the PR! I think we can just fix this by adding `./`

```suggestion
> See the [API documentation](./docs/api.md) for all endpoints.
```

But let me know if that doesn't work!

Reply:
Does not work.